pkg/build/android.go

```diff
@@ -75,22 +84,47 @@ func (a android) build(params Params) (ImageDetails, error) {
 		return details, fmt.Errorf("sysctl file is not supported for android cuttlefish images")
 	}
 
-	if err := a.runBuild(params.KernelDir, kernelConfig); err != nil {
-		return details, fmt.Errorf("failed to build kernel: %s", err)
+	var config string
+	var err error
+	// Find current android branch from build.config.constants file.
+	cmd := osutil.Command("/bin/sh", "-c", "grep -oP '(?<=BRANCH=android).*(?=-)' build.config.constants")
```

Comment:
I think the code could be simpler if you made use of the `params.Compiler` field to determine if kernel has be compiler with bazel.

Reply:
Good point, thanks! Fixed.